data/events/scripts/party.lua

```diff
@@ -9,3 +9,27 @@ end
 function Party:onDisband()
 	return true
 end
+
+function Party:onShareExperience(exp)
+	local sharedExperienceMultiplier = 1.20 --20%
+	local vocationsIds = {}
+
+	local vocationId = self:getLeader():getVocation():getBase():getId()
```

Comment:
That is a lot of indirections. Can someone with a large server benchmark?

Reply:
It is but I doubt anyone with a large server (if they are even using TFS 1.X) would just drop that in and test. I don't think that happens. I think we may find out in the long term and if that proves to be a problem, improve it.